etest/docker/container.py

```diff
@@ -16,7 +16,11 @@
 
 def commit(container: Container, tag: str, repository: str, *args, **kwargs):
     """Commit a Docker container."""
-    repo = repository.replace("=", "").replace("[", "").replace("]", "")
+    print(repository)
```

Comment:
Spurious print here.

Reply:
I wonder if we should make pre-commit catch those.

Reply:
That's a great idea.  I'd say add the hook if you know the one that does it.

Reply:
I cant find one that does it :/ however it might be easy to code so I might do that.